ban/__init__.py

```diff
@@ -0,0 +1,97 @@
+""" ban init file """
+import os
+import json
+import logging
+import traceback
+import requests
+import jsend
+import azure.functions as func
+from requests.models import Response
+from shared_code import common
+
+# map existing ban lookup fieldnames to socrata fieldnames
+ban_map = {
+    "BAN": "ban",
+    "OWNERS": "owners",
+    "BusinessName": "businessname",
+    "DBAName": "dbaname",
+    "StreetAddress": "streetaddress",
+    "City": "city",
+    "State": "state",
+    "PostalCd": "postalcd",
+    "LIN": "lin",
+    "BusstartDate": "busstartdate",
+    "BusEndDate": "busenddate",
+    "LocstartDate": "locstartdate",
+    "LocEndDate": "locenddate",
```

Comment:
Comparing the outputs with the existing API, we probably need to some light date conversion to matches with existing MM-DD-YYYY format.

Reply:
c875181